Source/CarthageKit/GitHub.swift

```diff
@@ -249,12 +322,12 @@ private func loadCredentialsFromGit() -> SignalProducer<BasicGitHubCredentials?,
 }
 
 
-internal func loadGitHubAuthorization() -> SignalProducer<String?, CarthageError> {
+internal func loadGitHubAuthorization(forURL URL: GitHubRepository.URL) -> SignalProducer<String?, CarthageError> {
 	let environment = NSProcessInfo.processInfo().environment
 	if let accessToken = environment["GITHUB_ACCESS_TOKEN"] as? String {
```

Comment:
The easiest solution would probably be to let this be a comma separated series of domains and tokens.

```
GITHUB_ACCESS_TOKEN = "github.com=XXXXXXXXXXXXX,myenterprise.foo=YYYYYYYYY"
```

We could still allow a token to be provided by itself for use with GitHub.com. This would give us backwards compatibility and make the simple case simple.

What do you think?